src/process.py

```diff
@@ -1,17 +1,19 @@
+# coding=utf-8
 """process.py - Process spawning module."""
+from __future__ import absolute_import
 
 import gc
 import subprocess
 
 
-class Process:
-
+class Process(object):
```

Comment:
same, I don't think `(object)` is necessary

Reply:
See above.

Reply:
agreed